morph_tool/converter.py

```diff
@@ -152,6 +152,19 @@ def single_point_sphere_to_circular_contour(neuron):
     _to_sphere(neuron)
 
 
+def soma_to_single_point(soma):
+    logger.info('Converting a cylindrical to a single point sphere')
+    assert soma.type == SomaType.SOMA_SIMPLE_CONTOUR
+    r0 = 0.5 * soma.diameters[:-1]
+    r1 = 0.5 * soma.diameters[1:]
+    h2 = np.sum((soma.points[:-1] - soma.points[1:]) ** 2., axis=1)
+
+    surface_area = np.sum(np.pi * (r0 + r1) * np.sqrt((r0 - r1) ** 2 + h2))
```

Comment:
eventually you could reuse neurom.morphmath.segment_area
or least explain that this is the sum of the conical frustums in a comment